src/compute/default_cuda_host_buffer.cpp

```diff
@@ -130,5 +130,16 @@ const void* default_cuda_host_buffer::get_data() const noexcept
     return m_block ? m_block->get_data() : nullptr;
 }
 
+device_buffer* default_cuda_host_buffer::get_device_accessible_alias() noexcept
+{
+    return nullptr;
+}
+
+const device_buffer* 
+default_cuda_host_buffer::get_device_accessible_alias() const noexcept
+{
+    return nullptr;
+}
+
```

Comment:
Why are both null? With both I mean this ones and the device ones. Shouldn't at least a couple of them (device or host) have something inside other than nullptr?

Reply:
Note that this method allows returning null and the caller should expect it. It is only here to allow fine-grained optimization.